README.md

```diff
@@ -1,6 +1,9 @@
 # Crystal
 
-[![Travis CI Build Status](https://travis-ci.org/crystal-lang/crystal.svg)](https://travis-ci.org/crystal-lang/crystal)
+![Linux CI](https://github.com/crystal-lang/crystal/workflows/Linux%20CI/badge.svg)
+![macOS CI](https://github.com/crystal-lang/crystal/workflows/macOS%20CI/badge.svg)
+![AArch64 CI](https://github.com/crystal-lang/crystal/workflows/AArch64%20CI/badge.svg)
+![Windows CI](https://github.com/crystal-lang/crystal/workflows/Windows%20CI/badge.svg)
```

Comment:
These would ideally be wrapped into links, such as https://github.com/crystal-lang/crystal/actions?query=workflow%3A%22Windows+CI%22+event%3Apush+branch%3Amaster

Reply:
One little issue is all fork runs are listed.